meilisearch-release-changelog/v0.30.0.md

```diff
@@ -0,0 +1,229 @@
+# v0.30.0 release changelogs
+
+<!-- The following line should NOT be put in the official release changelogs -->
+⚠️ Since this is a release candidate (RC), we do NOT recommend using it in a production environment. Something not working as expected? We welcome [bug reports](https://github.com/meilisearch/meilisearch/issues/new/choose) and [feedback about new features](https://github.com/meilisearch/product/discussions).
+
+This release removes index deployment downtime and adds new task management features. It also introduces new search parameters that facilitate the creation of numbered pagination interfaces.
+
+Thanks to Hacktoberfest, tons of updates and bug fixes have been done by the fantastic Meilisearch community. You were more present than ever and words cannot make justice tojust  how grateful we are for your support. A big thank you from the whole Meilisearch team! ❤️
+
+<!-- The following line should NOT be put in the official release changelogs -->
+📖 Meilisearch's official documentation will be ready for the official v0.30.0 release. Meanwhile, work-in-progress documentation is available [here]().
+
+# New features and improvements 🔥
+
+## Ease search results navigation with pagination
+
+You can now navigate through page selection when searching with Meilisearch using the API. This way, you will get the exact total number of pages and results Meilisearch has matched based on a query.
+
+For this, use `page` and `hitsPerPage` as query parameter instead of `offset` and `limit`:
+
+```bash
+curl \
+  -X POST 'http://localhost:7700/indexes/movies/search' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{ "q": "shifu", "page": "2", "hitsPerPage": "10" }'
+```
+
+```json
+{
+    "hits": [
+        // … 10 hits
+    ],
+    // …
+    "page": 2,
+    "hitsPerPage": 10,
+    "totalHits": 2100,
+    "totalPages": 210
+}
+```
+
+Done in #2601 by @ManyTheFish.
+
+## Cancel processing and enqueued tasks
+
+Meilisearch now allows you to cancel `enqueued` or `processing` tasks through a new API route, `/tasks/cancel`. This can be useful when you need to interrupt a process that is taking too much time to be completed.
```

Comment:
```suggestion
Meilisearch now allows you to cancel `enqueued` or `processing` tasks through a new API route, `/tasks/cancel`.

This prevents starting from scratch with a fresh meilisearch install nor having to restart the instance in the following cases.

- Cancel a task that is taking too much time to be processed.
- Cancel tasks enqueued by mistake.
```

Reply:
It's a little bit too long from my POV to do the list. We are doing a changelog here, not a blog post
Will try to incorporate some of your points!

Reply:
For me, talking about canceling a task does not mean we need to talk about preventing starting with a Meilisearch from scratch. Yes, it's the workaround, but it brings a complex idea in a changelog that tries to be simple, and exhaustive of the changes.
WDYT @guimachiavelli?

Reply:
(I made a small change so far, this is still something I can change once we agree on a final version)

Reply:
I've written: `Meilisearch now allows you to cancel `enqueued` or `processing` tasks through a new API route, `/tasks/cancel`. This can be useful when you need to interrupt a task that is taking too much time to be completed or enqueued by mistake.`

Reply:
> This prevents starting from scratch with a fresh meilisearch install

What is the relation between cancelling processing/enqueued tasks and fresh meilisearch installs?

Reply:
In the case people adding tasks too long to process (they want to cancel) or by mistake, people had to
- either wait for the task to be processed
- or kill their old instance, delete the data.ms, and start a new instance

Reply:
Yeah, I agree, then. Talking about fresh installs introduces too many topics that do not relate directly to cancelling tasks. The changelog should give a very quick rundown of a feature and describe simple uses to illustrate it.